### PR TITLE
habitat_package resource

### PR DESCRIPTION
**Pull request by clintoncwolfe**
part of #13 

Implements a singular package resource, which does not do anything useful - just name and version.

### CODE REVIEW

File: docs/resources/habitat_package.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+```ruby
+describe habitat_package(origin: 'core', name: 'httpd') do
+  its('pkg_id') { should cmp 'core/httpd/2.4.35/20190307151146' }
+end
+```
+
+### release
+
+String. The release number of the package, as assigned by the packager. These values are always strings, but are 14-digit timestamps. See also [version](#version).
```
Comment (mjingle) [inline docs/resources/habitat_package.md]:
```suggestion
`String`. The release number of the package, as assigned by the packager. These values are always strings, but are 14-digit timestamps. See also [version](#version).
```
I think I may have missed this formatting detail during the last sweep.

File: docs/resources/habitat_package.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+```ruby
+describe habitat_package(origin: 'core', name: 'httpd') do
+  its('origin') { should cmp 'core' }
+end
+```
+
+### pkg_id
+
+String. The full package identifier of the package, in the form `origin/name/version/release`.  See also [name](#name) and [version](#version).
```
Comment (mjingle) [inline docs/resources/habitat_package.md]:
```suggestion
`String`. The full package identifier of the package, in the form `origin/name/version/release`.  See also [name](#name) and [version](#version).
```
Another formatting miss on my part?

File: docs/resources/habitat_package.md (hunk 1 above)
Comment (clintoncwolfe) [reply docs/resources/habitat_package.md]: Thanks!

File: docs/resources/habitat_package.md (hunk 2 above)
Comment (clintoncwolfe) [reply docs/resources/habitat_package.md]: Thanks!

**Pull request merged**